src/beacon/app/routes/devices.py

```
@@ -0,0 +1,310 @@
+
+@router.get("/stats")
+async def get_comprehensive_device_stats(
+    *,
+    db: Session = Depends(get_db),
+    include_networks: bool = Query(True, description="Include network breakdown"),
+    include_categories: bool = Query(True, description="Include category breakdown"),
+    include_maintenance: bool = Query(True, description="Include maintenance info")
+):
+    total_devices = db.exec(select(func.count(Device.device_key))).first() or 0
+    active_devices = db.exec(
+        select(func.count(Device.device_key)).where(Device.is_active == True)
```

Ruff (0.12.2)

Avoid equality comparisons to `True`; use `Device.is_active:` for truth checks

Replace with `Device.is_active`

(E712)